intellij-plugin-verifier/verifier-intellij/src/main/java/com/jetbrains/pluginverifier/PluginVerifier.kt

```diff
@@ -125,7 +128,9 @@ class PluginVerifier(
         context.pluginResolver
       )
       groupMissingClassesToMissingPackages(context.compatibilityProblems, context.classResolver)
-      val (reportProblems, ignoredProblems) = partitionReportAndIgnoredProblems(context.compatibilityProblems, context)
+
+      val compatibilityProblems = context.compatibilityProblems + structureProblemsResolver.resolveCompatibilityProblems(context)
```

Comment:
Shouldn't the `UndeclaredKotlinK2CompatibilityMode` be excluded from `pluginStructureWarnings`. otherwise it will be reported twice as error and warning, if I don't miss anything.

Reply:
Corresponding resolver will now remove all such plugin structure warnings from the verification context. 
Added KDoc and unit test.